osquery/process/windows/process.cpp

```diff
@@ -356,10 +344,11 @@ std::shared_ptr<PlatformProcess> PlatformProcess::launchTestPythonScript(
   }
 
   auto pythonPath = *pythonEnv;
+  auto argv = L"python " + stringToWstring(args);
 
   std::shared_ptr<PlatformProcess> process;
   if (::CreateProcessW(stringToWstring(pythonPath).c_str(),
```

Comment:
I have a questions about Win32 API calls and its names.
Should the `W` form of the function be used explicitly if possible or there is a correct macro that expands `CreateProcess` as `CreateProcessW` across all project?

Reply:
Yes, Windows functions should be used explicitly when possible. Windows SDK already provides a macro to expand the name of the function based on the UNICODE variable - see [here](https://learn.microsoft.com/en-us/windows/win32/intl/conventions-for-function-prototypes)

Reply:
Yes, Win SDK define a lot of macro that expand to A/W functions.
But it relies on UNICODE definition and can even break some code (e.g. if you have method of function with name `SendMessage`)
That is why some projects ban these macroses and force explicit function names usage.